Makefile

```diff
@@ -1,12 +1,34 @@
-all: build_native
+# Go parameters
+GOCMD=go
+GOBUILD=$(GOCMD) build
+GOCLEAN=$(GOCMD) clean
+GOTEST=$(GOCMD) test
+BINARY_NAME=kubeaudit
+BINARY_UNIX=$(BINARY_NAME)_unix
 
-build_native:
-	go build -o kubeaudit .
+all: test build
+
+build:
+	$(GOBUILD) -o $(BINARY_NAME) -v
 
 test:
-	go test -cover ./cmd/... .
+	$(GOTEST) -cover ./... .
 
 clean:
-	/bin/rm -v kubeaudit
+	$(GOCLEAN)
+	rm -f $(BINARY_NAME)
+	rm -f $(BINARY_UNIX)
+
+run:
+	$(GOBUILD) -o $(BINARY_NAME) -v ./...
+	./$(BINARY_NAME)
+
+
+# Cross Compilation
+build-linux:
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 $(GOBUILD) -o $(BINARY_UNIX) -v
+
+docker-build:
+	docker run --rm -it -v "$(GOPATH)":/go -w /go/src/github.com/Shopify/kubeaudit golang:1.9 go build -o "$(BINARY_UNIX)" -v
```

Comment:
What does the second `-v` do?

Reply:
Second -v is for verbose build it's a go build flag